sdks/router-sdk/src/entities/trade.ts

```diff
@@ -239,6 +273,11 @@ export class Trade<TInput extends Currency, TOutput extends Currency, TTradeType
     return new Percent(outputCurrency.wrapped.buyFeeBps.toNumber(), 10000)
   }
 
+  private isWrappedNative(currency: Currency): boolean {
```

Comment:
i guess this would ideally go in the Currency class? is there a reason it cant?

Reply:
ideally yeah maybe it lives in the `Token` class but it would be a different PR bc its in sdk-core